client/src/assets/arrow.svg

```diff
@@ -3,6 +3,6 @@
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 width="100px" height="100px" viewBox="0 0 100 100" enable-background="new 0 0 100 100" xml:space="preserve">
-<path fill="none" stroke="#000000" stroke-width="6" stroke-miterlimit="10" d="M61.432,36.095V19.396l28.824,27.633L61.432,74.662
+<path fill="none" stroke="#0055CC" stroke-width="7" stroke-miterlimit="10" d="M61.432,36.095V19.396l28.824,27.633L61.432,74.662

```

Comment:
I'm super impressed! You actually work on your svg edits and creation...

Reply:
Oh, actually I did all of this in Adobe Illustrator. If you save your vectors as SVG, it's really just all code. 
AHEM I MEAN yes I coded it all out